packages/pn-pa-webapp/src/pages/__test__/Dashboard.page.a11y.test.tsx

```diff
@@ -45,7 +45,7 @@ describe('Dashboard Page - accessibility tests', () => {
       .onGet(
         NOTIFICATIONS_LIST({
           startDate: formatToTimezoneString(tenYearsAgo),
-          endDate: formatToTimezoneString(getNextDay(today)),
+          endDate: formatToTimezoneString(getEndOfDay(today)),
```

Comment:
today constant is calculated using getEndOfDay(new Date()), so this usage of getEndOfDay is useless.
Remove it from this file